content/developer/reference/backend/views.rst

```diff
@@ -178,6 +178,17 @@ an impact on all view types.
 
 .. todo:: Views main content section, with field, group & separator ?
 
+* ``class``
+  optional string containing some classes to set on the root node of the view.
+
+* ``js_class``
+  optional component class to use to render the view.
+  If set, the ``js_class`` value will also be used to replace the placeholder in
```

Comment:
not clear, in my opinion. Here is a suggestion:
```
If set, the ``js_class`` attribute will also be used to apply a css class to the
root node of the view. For example, if ...```

Reply:
done